### ISSUE TITLE
some responses give a expires header which lies in the past

### ISSUE THREAD

**Issue by devnulli** [ccpgames/esi-issues#637]
# Bug
some responses give back an expires header which lies in the past
to be exact, it mostly lies in the past day, at 11:05 am GMT

so for example, a request sent today on Nov 5 in the afternoon (say 16:00 GMT) sometimes has an expires header with the value of
'Sun, 05 Nov 2017 11:05:00 GMT'

Responding with a past date in the Expired header (earlier than the Date header value) makes no sense and would be a sign of some serious misconfiguration. 
Clients would treat such response as "already expired" and not cache it. 

This happens often, but not everytime. It's clearly reproducable if you want a test driver to reproduce it, contact me.

### Request
from what i found out by now, 
it happens often in universe routes.
it happens, but rarely, in market routes

### Response
in the headers:

https://ibb.co/cbwVfw

### Expected
response header value expires > header value of date
response header value not in the past

**Comment by lukasni** [comment]
11:05:00 GMT sounds like the static `/universe/*` routes that expire daily after downtime. Is it possible those aren't being invalidated properly by the CDN?

**Comment by devnulli** [comment]
I'm at work now, but i will try to find out more. i don't think it's the Invalidation mechanism, because the same request sometimes has a correct expiry date, then minutes later has a wrong expiry date, then, minutes later, has a correct expiry date again.

Maybe, if there is load balancing, and one of the hit servers is misconfigured. (One day in the past??) 

I can reprodue it by iterating all regions->constellations->Systems->stargates and check the expiry Dates, there is always at least a dozen of requests that have that Problem

I will get deeper in this in the evening.

**Comment by aquarhead** [comment]
Probably related #487 

@a-tal ^

**Comment by devnulli** [comment]
@aquarhead  that #487 definitely happened to me yesterday, as well as that #194

**Comment by lukasni** [comment]
I have not seen #487 happen on any of the market endpoints since the fix went live, as in, specifically related to different pages having different expiry dates.

Will do some tests on the 11:05 expiry rates to see what I get here

**Comment by lukasni** [comment]
Okay, ran some tests on `/v1/dogma/attributes/{attribute_id}`. Got 19/2385 requests with an `Expires` header of `Mon, 06 Nov 2017 11:05:00 GMT`.

Requesting any of those a second time resulted in a correct `Expire` header of `Tue, 06 Nov 2017 11:05:00 GMT`

Overall seems to confirm what @devnulli has been encountering. I haven't seen anything similar on `/markets/{region_id/orders/` since the fix for #487 went live, so can't confirm there. I am able to reproduce the above issue with `/markets/groups/{group_id}/` though, which is another of those `11:05` expiry endpoints.

The following list of endpoint have that expiry and are likely to be affected in the same way:

```
'/v1/dogma/attributes/'
'/v1/dogma/attributes/{attribute_id}/'
'/v1/dogma/effects/'
'/v2/dogma/effects/{effect_id}/'
'/v1/fw/wars/'
'/v1/fw/stats/'
'/v1/fw/leaderboards/'
'/v1/fw/leaderboards/characters/'
'/v1/fw/leaderboards/corporations/'
'/v1/corporations/{corporation_id}/fw/stats/'
'/v1/characters/{character_id}/fw/stats/'
'/v1/loyalty/stores/{corporation_id}/offers/'
'/v1/markets/groups/'
'/v1/markets/groups/{market_group_id}/'
'/v1/opportunities/groups/'
'/v1/opportunities/groups/{group_id}/'
'/v1/opportunities/tasks/'
'/v1/opportunities/tasks/{task_id}/'
'/v1/universe/planets/{planet_id}/'
'/v3/universe/systems/{system_id}/'
'/v1/universe/systems/'
'/v3/universe/types/{type_id}/'
'/v1/universe/types/'
'/v1/universe/groups/'
'/v1/universe/groups/{group_id}/'
'/v1/universe/categories/'
'/v1/universe/categories/{category_id}/'
'/v1/universe/races/'
'/v1/universe/factions/'
'/v1/universe/bloodlines/'
'/v1/universe/regions/'
'/v1/universe/regions/{region_id}/'
'/v1/universe/constellations/'
'/v1/universe/constellations/{constellation_id}/'
'/v1/universe/moons/{moon_id}/'
'/v1/universe/stargates/{stargate_id}/'
'/v1/universe/graphics/'
'/v1/universe/graphics/{graphic_id}/'
'/v1/universe/stars/{star_id}/'
```

**Comment by devnulli** [comment]
is there any Information in those responses that would come in handy to find the error(s)?

im pretty sure that i ran into the same issue with market/{region_id}/orders?page={n} yesterday, 
and im pretty sure i had one market order in two pages after that. i could probably save that request/response data when i encounter that again, but only if it is of any help. 

also, it seems to occur very often, and then another time´very rarely, so could it depend on System load or time of day?

**Comment by lukasni** [comment]
@devnulli Are you sure you didn't hit cache rollover while fetching pages? Like, pages 1-n having one set of Expires-headers, then n-max having another. That happens if cache rolls over while you're fetching, you'll have to re-request the ones with the old header.

**Comment by devnulli** [comment]
yes i'm sure, because the i only hit the breakpoint when expiry < Datetime.Now 
but it happened only ONCE so far, so i suggest we ignore that, and if i encounter that again, i open another issue

**Comment by a-tal** [comment]
can I get the full set of headers for one of these requests?

**Comment by lukasni** [comment]
`/v1/dogma/attributes/4/`

```
'Access-Control-Allow-Credentials': 'true'
'Access-Control-Allow-Headers': 'Content-Type,Authorization,X-User-Agent'
'Access-Control-Allow-Methods': 'GET,OPTIONS'
'Access-Control-Allow-Origin': '*'
'Access-Control-Expose-Headers': 'Content-Type,Warning'
'Access-Control-Max-Age': '600'
'Content-Length': '190'
'Content-Type': 'application/json'
'Date': 'Mon, 06 Nov 2017 13:43:34 GMT'
'ETag': 'W/"a1227e955645c9ef47390e7745922cd2"'
'Expires': 'Mon, 06 Nov 2017 11:05:00 GMT'
'Last-Modified': 'Sun, 05 Nov 2017 12:22:13 GMT'
'Strict-Transport-Security': 'max-age=31536000'
'X-Esi-Error-Limit-Remain': '100'
'X-Esi-Error-Limit-Reset': '9'
'Via': '1.1 google'
'Cache-Control': 'public'
'Age': '0'
'Alt-Svc': 'clear'
```

and after fetching it again:

```
'Access-Control-Allow-Credentials': 'true'
'Access-Control-Allow-Headers': 'Content-Type,Authorization,X-User-Agent'
'Access-Control-Allow-Methods': 'GET,OPTIONS'
'Access-Control-Allow-Origin': '*'
'Access-Control-Expose-Headers': 'Content-Type,Warning'
'Access-Control-Max-Age': '600'
'Cache-Control': 'public'
'Content-Length': '190'
'Content-Type': 'application/json'
'Date': 'Mon, 06 Nov 2017 14:09:05 GMT'
'Etag': 'W/"a1227e955645c9ef47390e7745922cd2"'
'Expires': 'Tue, 07 Nov 2017 11:05:00 GMT'
'Last-Modified': 'Mon, 06 Nov 2017 14:09:05 GMT'
'Strict-Transport-Security': 'max-age=31536000'
'X-Esi-Error-Limit-Remain': '100'
'X-Esi-Error-Limit-Reset': '55'
'Via': '1.1 google'
'Alt-Svc': 'clear'
```

**Comment by a-tal** [comment]
thanks. taking a look at this now

**Comment by devnulli** [comment]
ok, so i tried some things, and could not reproduce the issue with the market/{regionid}/orders route. seems that i was wrong on that

**Comment by a-tal** [comment]
I have a fix in staging, will leave it there overnight and take a look in the morning

**Comment by devnulli** [comment]
the problem disappeared

**Comment by a-tal** [comment]
this should be resolved as part of esi `0.7.2`.

**Comment by devnulli** [comment]
It's happening again..
15.11.2017 18:20:48, Negative cache time at https://esi.tech.ccp.is/latest/universe/regions/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 11
	Age = 0
	Alt-Svc = clear
	Content-Length = 901
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:52 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:10:33 GMT
	Via = 1.1 google
15.11.2017 18:20:52, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000001/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 48
	Age = 0
	Alt-Svc = clear
	Content-Length = 550
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:55 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:06 GMT
	Via = 1.1 google
15.11.2017 18:20:52, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000019/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 48
	Age = 0
	Alt-Svc = clear
	Content-Length = 1104
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:56 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:13 GMT
	Via = 1.1 google
15.11.2017 18:20:53, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000075/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 46
	Age = 0
	Alt-Svc = clear
	Content-Length = 1336
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:56 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:31 GMT
	Via = 1.1 google
15.11.2017 18:20:53, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000007/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 46
	Age = 0
	Alt-Svc = clear
	Content-Length = 844
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:57 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:08 GMT
	Via = 1.1 google
15.11.2017 18:20:53, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000102/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 40
	Age = 0
	Alt-Svc = clear
	Content-Length = 906
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:57 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:41 GMT
	Via = 1.1 google
15.11.2017 18:20:54, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000107/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 45
	Age = 0
	Alt-Svc = clear
	Content-Length = 842
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:58 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:43 GMT
	Via = 1.1 google
15.11.2017 18:20:55, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000084/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 45
	Age = 0
	Alt-Svc = clear
	Content-Length = 770
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:58 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:34 GMT
	Via = 1.1 google
15.11.2017 18:20:55, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000030/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 43
	Age = 0
	Alt-Svc = clear
	Content-Length = 946
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:20:59 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:17 GMT
	Via = 1.1 google
15.11.2017 18:20:56, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000112/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 45
	Age = 0
	Alt-Svc = clear
	Content-Length = 614
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:00 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:44 GMT
	Via = 1.1 google
15.11.2017 18:20:56, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000109/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 44
	Age = 0
	Alt-Svc = clear
	Content-Length = 895
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:00 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:43 GMT
	Via = 1.1 google
15.11.2017 18:20:56, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000037/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 43
	Age = 0
	Alt-Svc = clear
	Content-Length = 863
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:00 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:20 GMT
	Via = 1.1 google
15.11.2017 18:20:58, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000005/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 42
	Age = 0
	Alt-Svc = clear
	Content-Length = 1336
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:02 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:08 GMT
	Via = 1.1 google
15.11.2017 18:20:58, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000082/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 41
	Age = 0
	Alt-Svc = clear
	Content-Length = 1289
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:02 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:34 GMT
	Via = 1.1 google
15.11.2017 18:20:59, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000056/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 40
	Age = 0
	Alt-Svc = clear
	Content-Length = 972
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:03 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:26 GMT
	Via = 1.1 google
15.11.2017 18:20:59, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000044/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 40
	Age = 0
	Alt-Svc = clear
	Content-Length = 575
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:03 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:23 GMT
	Via = 1.1 google
15.11.2017 18:21:00, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000040/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 39
	Age = 0
	Alt-Svc = clear
	Content-Length = 880
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:04 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:21 GMT
	Via = 1.1 google
15.11.2017 18:21:01, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000012/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 38
	Age = 0
	Alt-Svc = clear
	Content-Length = 1349
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:04 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:11 GMT
	Via = 1.1 google
15.11.2017 18:21:01, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000087/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	Vary = Accept-Encoding
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 37
	Age = 0
	Alt-Svc = clear
	Content-Length = 1305
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:05 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:35 GMT
	Via = 1.1 google
15.11.2017 18:21:01, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000015/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 37
	Age = 0
	Alt-Svc = clear
	Content-Length = 914
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:05 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:12 GMT
	Via = 1.1 google
15.11.2017 18:21:03, Negative cache time at https://esi.tech.ccp.is/latest/universe/systems/30000074/?datasource=tranquility	Access-Control-Allow-Credentials = true
	Access-Control-Allow-Headers = Content-Type,Authorization,X-User-Agent
	Access-Control-Allow-Methods = GET,OPTIONS
	Access-Control-Allow-Origin = *
	Access-Control-Expose-Headers = Content-Type,Warning
	Access-Control-Max-Age = 600
	Content-Language = en-us
	Strict-Transport-Security = max-age=31536000
	X-Esi-Error-Limit-Remain = 100
	X-Esi-Error-Limit-Reset = 35
	Age = 0
	Alt-Svc = clear
	Content-Length = 956
	Cache-Control = public
	Content-Type = application/json
	Date = Wed, 15 Nov 2017 17:21:07 GMT
	Expires = Wed, 15 Nov 2017 11:05:00 GMT
	ETag = W/"12dcf8b2718e042f3b5cbc0d0a34f509"
	Last-Modified = Tue, 14 Nov 2017 11:11:31 GMT
	Via = 1.1 google

**Comment by Shirrath** [comment]
/v1/markets/{regionid}/history is also a frequent offender:
[markethistoryfailure.txt](https://github.com/ccpgames/esi-issues/files/1475844/markethistoryfailure.txt)
